### ISSUE TITLE
Javadoc

### ISSUE THREAD

**Issue by DigitalSmile** [JeremyWildsmith/bazaarbot-java#9]
We need javadoc to be added for all methods. There are parts not obvious at all...

**Comment by JeremyWildsmith** [comment]
I agree; as well I think we need a wiki page to go over how the algorithm(s) work in the simulation since it has since been changed from the paper. I'll look into writing the wiki pages.

**Comment by DigitalSmile** [comment]
I guess we need to create a wiki only when we finish the basic refactoring. Right now it is more like a mess...

**Comment by JeremyWildsmith** [comment]
Hey, so I am working on the Wiki right now; below is a visualization/report I am going to write a tutorial on creating in the wiki. It will go through adding the project as a maven dependency from the central repo, setting up the economy (Classes / Commodities) and export a CSV report and finally importing into google docs to visualize:
https://docs.google.com/spreadsheets/d/1x14xefLA-DiRlWZP0xs9ewrpkz60BKlgs_DQwVIFd_k/edit?usp=sharing
